src/components/Puppies.jsx

```diff
@@ -2,35 +2,35 @@ import React from "react";
 import { useContext } from "react";
```

Comment:
```suggestion
import React, { useContext } from "react";
```

Suggest merging two same-source import into one.